src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from 'react-i18next'
+import { Box, Typography } from '@mui/material'

```

Comment:
Use imports like this, for all mui components:
import Box from '@mui/material/Box'

Reply:
fixed